### DISCUSSION TITLE
Is it possible to install eza on CentOS 7?

### DISCUSSION THREAD

**Discussion by mlopezcoria** [eza-community/eza#965] [Q&A, Howto, Help]
Hello,

I can't find instructions/guide to install eza neither on CentOS 7 nor CentOS/AlmaLinux 8. Is that possible?

Regards,
Mauricio

**Comment by MartinFillon** [comment]
Well you can always run cargo install eza, or compile it yourself. For more info see [this](https://github.com/eza-community/eza/blob/main/INSTALL.md) file.
